charts/node/README.md

```diff
@@ -68,18 +68,26 @@ helm install kusama-node parity/node --set node.chainDataSnapshotUrl=https://ksm
+| Parameter                          | Description                                                                                            | Default                   |
+|------------------------------------|--------------------------------------------------------------------------------------------------------|---------------------------|
+| `image.repository`                       | Node image name                                                                                        | `parity/polkadot`   |
+| `image.tag`                              | Node image tag                                                                                         | `latest`            |
+| `image.pullPolicy`                       | Node image pull policy                                                                                 | `Always`            |
+| `initContainer.image.repository`         | Download-chain-snapshot init container image name                                                      | `crazymax/7zip`     |
+| `initContainer.image.tag`                | Download-chain-snapshot init container image tag                                                       | `latest`            |
+| `googleCloudSdk.image.repository`        | Sync-chain-gcs init container image name                                                               | `google/cloud-sdk`  |
+| `googleCloudSdk.image.tag`               | Sync-chain-gcs init container image tag                                                                | `slim`              |
+| `googleCloudSdk.serviceAccountKey`       | Service account key (JSON) to inject into the Sync-chain-gcs init container using a Kubernetes secret  | `nil`               |
+| `ingress.enabled`                        | If true, creates an ingress                                                                            | `false`             |
+| `ingress.annotations`                    | Annotations to add to the ingress (key/value pairs)                                                    | `{}`                |
+| `ingress.rules`                          | Set rules on the ingress                                                                               | `[]`                |
+| `ingress.tls`                            | Set TLS configuration on the ingress                                                                   | `[]`                |
+| `podSecurityContext`                     | Set the pod security context for the substrate node container                                          | `{ runAsUser: 1000, runAsGroup: 1000, fsGroup: 1000 }`|
+| `sidecar.jaegerAgent.enabled`            | If true, enabled Jaeger agent as sidecar                                                               | `false`             |
```

Comment:
I would use the following variable name: `node.tracing.enabled` to enable tracing on the node and based on that activate the jaeger agent and use the properties such as `jaegerAgent.collector.url`...

Reply:
Addressed this with the latest commit (a5ad3c9df73)